Limited download speed on MacOS Mojave

When I download files from a server behind the VPN using openfortivpn my download speeds are capped around 2.5mb/sec. When I perform the same download behind the official Fortinet client, I can maximize my connection speed (around 10mb/sec). 

Any ideas what might be happening there? The problem is reproducible on different MacOS laptops when downloading the same files.

> @JPlanche now that I have merged the pull request you can simply download the current master and compile that one.

@mrbaseman  I have these brew formulaes installed: automake autoconf openssl@1.0 pkg-config.
I exported the LDFLAGS and CPPFLAGS variables.
I have openssl in the PATH:
`$ openssl version
OpenSSL 1.0.2s  28 May 2019`
But configure fails on:
`checking for libssl >= 0.9.8 libcrypto >= 0.9.8... no
configure: error: Cannot find OpenSSL 0.9.8 or higher.`

Sorry, I'm not very used to compilation. :-) I did a search but didn't find any clue...

--- a/src/tunnel.c
+++ b/src/tunnel.c
@@ -232,7 +232,7 @@ static int pppd_run(struct tunnel *tunnel)
 		} else {
 			static const char *const v[] = {
 				ppp_path,
-				"115200", // speed
+				"230400", // speed
 				":192.0.2.1", // <local_IP_address>:<remote_IP_address>
 				"noipdefault",
 				"noaccomp",

